test/fw/ptl/lib/ptl_server.py

```diff
@@ -1860,7 +1860,14 @@ def create_moms(self, name=None, attrib=None, num=1, delall=True,
             except PbsManagerError as e:
                 rc = e.rc
             if rc:
-                if len(self.status(NODE)) > 0:
+                node_length = 0
+                try:
+                    node_length = len(self.status(NODE))
+                except PbsStatusError as err:
+                    if "Server has no node list" not in err.msg[0]:
+                        self.logger.error("node sttus err is " + str(err))
```

Comment:
This isn't a very informative error message.  Could you try to make it a little more explanatory to what the problem is?